cypress/fixtures/monster.json

```diff
@@ -1,149 +1 @@
re, it is grappled (escape DC 20). Until this grapple ends, the target is restrained, and the tarrasque can't bite another target.","attack_bonus":19,"damage":[{"damage_type":{"index":"piercing","name":"Piercing","url":"/api/damage-types/piercing"},"damage_dice":"4d12+10"}]},{"name":"Claw","desc":"Melee Weapon Attack: +19 to hit, reach 15 ft., one target. Hit: 28 (4d8 + 10) slashing damage.","attack_bonus":19,"damage":[{"damage_type":{"index":"slashing","name":"Slashing","url":"/api/damage-types/slashing"},"damage_dice":"4d8+10"}]},{"name":"Horns","desc":"Melee Weapon Attack: +19 to hit, reach 10 ft., one target. Hit: 32 (4d10 + 10) piercing damage.","attack_bonus":19,"damage":[{"damage_type":{"index":"piercing","name":"Piercing","url":"/api/damage-types/piercing"},"damage_dice":"4d10+10"}]},{"name":"Tail","desc":"Melee Weapon Attack: +19 to hit, reach 20 ft., one target. Hit: 24 (4d6 + 10) bludgeoning damage. If the target is a creature, it must succeed on a DC 20 Strength saving throw or be knocked prone.","attack_bonus":19,"damage":[{"damage_type":{"index":"bludgeoning","name":"Bludgeoning","url":"/api/damage-types/bludgeoning"},"damage_dice":"4d6+10"}]},{"name":"Frightful Presence","desc":"Each creature of the tarrasque's choice within 120 feet of it and aware of it must succeed on a DC 17 Wisdom saving throw or become frightened for 1 minute. A creature can repeat the saving throw at the end of each of its turns, with disadvantage if the tarrasque is within line of sight, ending the effect on itself on a success. If a creature's saving throw is successful or the effect ends for it, the creature is immune to the tarrasque's Frightful Presence for the next 24 hours.","dc":{"dc_type":{"index":"wis","name":"WIS","url":"/api/ability-scores/wis"},"dc_value":17,"success_type":"none"},"damage":[]},{"name":"Swallow","desc":"The tarrasque makes one bite attack against a Large or smaller creature it is grappling. If the attack hits, the target takes the bite's damage, the target is swallowed, and the grapple ends. While swallowed, the creature is blinded and restrained, it has total cover against attacks and other effects outside the tarrasque, and it takes 56 (16d6) acid damage at the start of each of the tarrasque's turns.\nIf the tarrasque takes 60 damage or more on a single turn from a creature inside it, the tarrasque must succeed on a DC 20 Constitution saving throw at the end of that turn or regurgitate all swallowed creatures, which fall prone in a space within 10 feet of the tarrasque. If the tarrasque dies, a swallowed creature is no longer restrained by it and can escape from the corpse by using 30 feet of movement, exiting prone.","damage":[]}],"legendary_actions":[{"name":"Attack","desc":"The tarrasque makes one claw attack or tail attack."},{"name":"Move","desc":"The tarrasque moves up to half its speed."},{"name":"Chomp (Costs 2 Actions)","desc":"The tarrasque makes one bite attack or uses its Swallow."}],"url":"/api/monsters/tarrasque"}
```

Comment:
Yeah, old json had a different monster. Went with this one because it has most of the traits possible.